python/README.md

```diff
@@ -51,17 +51,36 @@ Detailed instructions for how to install the complete CUDA toolkit on different
 operating systems can be found in the [CUDA
 documentation](https://docs.nvidia.com/cuda/).
 
-If you have several GPUs available but no MPI installation yet, we recommend
-taking a look at the [OpenMPI documentation](https://docs.open-mpi.org/)
-and installing [mpi4py](https://mpi4py.readthedocs.io/).
-On Ubuntu 22.04, for example, the following commands install the necessary MPI
-libraries:
+If you have several GPUs available but no MPI with CUDA support installation
+(CUDA-aware version of Open-MPI, MVAPICH, or MPICH) yet,
+we recommend taking a look at the [NVIDIA HPC-X](https://developer.nvidia.com/networking/hpc-x/)
+and also installing [mpi4py](https://mpi4py.readthedocs.io/).
+
+To install [NVIDIA HPC-X](https://developer.nvidia.com/networking/hpc-x/),
+for example, please download the appropriate archive for your system OS,
+then follow the step described in its
+[documentation](https://docs.nvidia.com/networking/display/hpcxv217/installing+and+loading+hpc-x)
+to install the package.
+Typically, the installation simply involves extracting the package
+and executing the install script:
+
+```console
+tar -xvjf hpcx-xxx-xxx.tbz
+cd hpcx-xxx-xxx
+source hpcx-init.sh
+hpcx_load
+```
+
+On Ubuntu 22.04, [mpi4py](https://mpi4py.readthedocs.io/) can be installed with:
 
 ```console
-sudo apt-get update && sudo apt-get install -y libopenmpi-dev libpython3-dev gcc
+sudo apt-get update && sudo apt-get install -y libpython3-dev gcc
 python3 -m pip install mpi4py
 ```
 
+Additional information about using the multi-GPU simulator backend
+can be found in the [documentation](https://nvidia.github.io/cuda-quantum/latest/using/simulators.html#multi-node-multi-gpu).
```

Comment:
Lets link only to the page that directs people to the docs with the different versions. Alternatively, you would need to link to the concrete docs version for this version of the wheel, and we would need to update the link for every release - there is a high risk that this will be forgotten, so I'd rather avoid it.
```suggestion
can be found in the [documentation][cudaq_docs_releases].
```